### ISSUE TITLE
Too much white-space in portrait mode

### ISSUE THREAD

**Issue by pimlie** [nuxt/nuxtjs.org#229]
### Version

[v1.1.0](https://github.com/nuxtjs.org/releases/tag/v1.1.0)

### Reproduction link

[https://nuxtjs.org](https://nuxtjs.org)

### Steps to reproduce

See screenshot of examples page, the main content container seems to take up the full height of the screen which looks a bit strange in portrait mode (screenshot is already scrolled down to bottom)

### What is expected ?

na

### What is actually happening?

na

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxtjs.org/issues/c172">#c172</a>)</em></sub></div>

**Comment by bdrtsky** [comment]
I can't reproduce this. Can you still confirm this issue @pimlie

**Comment by pimlie** [comment]
@bdrtsky Yep, its still happens. Note it only happens on a portrait screen, not landscape:
![image](https://user-images.githubusercontent.com/1067403/78304371-dd555280-753e-11ea-89b4-e7dbdc581e6e.png)

**Comment by codetheorist** [comment]
Confirming this is still an issue, in Chrome emulating a portrait iPad Pro

**Comment by bdrtsky** [comment]
<img width="1131" alt="Screenshot 2020-06-16 at 19 08 11" src="https://user-images.githubusercontent.com/45076741/84799302-c2a25e00-b004-11ea-95de-8e419285fc50.png">

Can't reproduce.

**Comment by codetheorist** [comment]
@bdrtsky It only happens in portrait mode

![nuxtjs org_guide](https://user-images.githubusercontent.com/22813671/85115903-59913500-b214-11ea-9f31-b6e8829fd767.png)
![nuxtjs org_guide (5)](https://user-images.githubusercontent.com/22813671/85115906-5ac26200-b214-11ea-9b80-6324f98093ee.png)

**Comment by codetheorist** [comment]
[Example Video](https://youtu.be/x4NKI_wow3k)

**Comment by codetheorist** [comment]
The only NOTICEABLE occurrence of this bug I can currently find is on the Typescript page however, some of the other pages do seem to include a relatively large section of unnecessary blank-space between the page content/menu and the footer area.

The bug does not happen on pages with long content.

**Comment by bdrtsky** [comment]
Thanks, found the issue. I am on it.
